CHANGELOG.md

```diff
@@ -8,11 +8,17 @@ Update `entrypoint.sh` to support the new version of Airflow.
 
 BREAKING CHANGES:
 
+`ENABLE_AIRFLOW_RBAC_SETUP_AUTH` takes on a different meaning for Airflow V2,
+as only the RBAC UI is available in V2 while RBAC and non-RBAC UI is available
+in V1. It just means to creating an user for V2 with the given env var values.
```

Comment:
Nit: Weird grammar.

Do you mean: "For V2, set to "true", to automatically create the user?

Reply:
Fixed, thanks for the catch.